maven/README.md

```diff
@@ -157,49 +157,71 @@ spec:
         #!/usr/bin/env bash
         
         [[ -f /.m2/settings.xml ]] && \
-        echo 'using already existing /.m2/settings.xml' && \
+        echo 'using existing /.m2/settings.xml' && \
         cat /.m2/settings.xml && exit 0
 
-        [[ -n '$(inputs.params.MAVEN_MIRROR_URL)' ]] && \
         cat > /.m2/settings.xml <<EOF
         <settings>
           <mirrors>
-            <mirror>
-              <id>mirror.default</id>
-              <name>mirror.default</name>
-              <url>$(inputs.params.MAVEN_MIRROR_URL)</url>
-              <mirrorOf>*</mirrorOf>
-            </mirror>
+            <!-- ### configured mirrors ### -->
           </mirrors>
           <proxies>
-            <proxy>
-              <id>proxy.default</id>
-              <active>true</active>
-              <protocol>$(inputs.params.PROXY_PROTOCOL)</protocol>
-              <username>$(inputs.params.PROXY_USER)</username>
-              <password>$(inputs.params.PROXY_PASSWORD)</password>
-              <host>$(inputs.params.PROXY_HOST)</host>
-              <port>$(inputs.params.PROXY_PORT)</port>
-              <nonProxyHosts>$(inputs.params.NON_PROXY_HOST)</nonProxyHosts>
-            </proxy>
+            <!-- ### configured http proxy ### -->
```

Comment:
maybe make it a bit clearer by formatting that this part will be replace by using something like in the suggestion below. This would also keep a marker in the XML to let the user know that this part is auto-generated


```
<!-- The proxies added here are generated from environment variables. Don't change. -->
<!-- [[ PROXY_FROM_ENV ]] -->
```

and then also remove the <!-- [[PROXY_FROM_ENV --> when no proxy env var is specified.